src/ScottPlot/DataGen.cs

```diff
@@ -9,6 +9,8 @@
 using System.Text;
 using System.Threading.Tasks;
 
+// TODO: Need to inspect Random instantiation because some of them is seed (new Random(0)), some not (new Random())
```

Comment:
This question is open for discussion. I'm not sure whether `new Random(0)` should be replaced. Maybe this is by design?

```csharp
public static double[] NoisyLinear(Random? rand, int pointCount = 100, double slope = 1, double offset = 0, double noise = 0.1)
```
For example if call it twice I'll get same results. This may be problem

Reply:
This is the intended behavior, but I recognize this is unclear. Soon I will mark the `DataGen` class obsolete and replace it with with a `Generate` class which will have better method names and documentation to make this easier to understand.